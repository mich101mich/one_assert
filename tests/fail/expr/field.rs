fn main() {
    struct Bob {
        age: u32,
    }
    let bob = Bob { age: 35 };
    one_assert::assert!(bob.age);
}
