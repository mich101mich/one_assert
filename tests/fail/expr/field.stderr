error[E0308]: mismatched types
 --> tests/fail/expr/field.rs:6:25
  |
6 |     one_assert::assert!(bob.age);
  |                         ^^^^^^^ expected `bool`, found `u32`
